bitex/api/REST/rest.py

```diff
@@ -9,6 +9,7 @@
 # Import Homebrew
 from bitex.api.base import BaseAPI
 
+requests.adapters.DEFAULT_RETRIES = 10
```

Comment:
This should not be set in here. Let the user decide how often they'd like to retry.

Reply:
It's in the last error push, I had reverted.
why it exist here?